xformers/triton/dropout.py

```diff
@@ -166,15 +169,22 @@ def dropout(
     Optionally add a bias, the computation will be fused.
     """
 
+    assert p < 1.0, f"We don't want to drop all the values, most probably {p}"
```

Comment:
PyTorch supports this case, so if we want our dropout to be a drop-in replacement to PyTorch's implementation it would be good to support this as well.

Reply:
fixed, incoming update, thanks for the catch !